### PR TITLE
Gives devil saturation on receiving a soul

### PR DESCRIPTION
**Pull request by Time-Green**
Devils have high-difficulty not starving because they can only materialize near potential signers, and not food.

This PR makes him go to well fed upon buying a soul and adds a (not so) fancy message.

### CODE REVIEW

**Comment by drynwyn** [comment]
It should say "satiated", not "saturated".

**Comment by timkoster1** [comment]
Ah, thank you, English is not my first language

File: code/game/gamemodes/devil/devilinfo.dm [hunk 1]
```diff
@@ -138,9 +138,12 @@ var/global/list/lawlorify = list (
 	return pick(BANISH_WATER, BANISH_COFFIN, BANISH_FORMALDYHIDE, BANISH_RUNES, BANISH_CANDLES, BANISH_DESTRUCTION, BANISH_FUNERAL_GARB)
 
 /datum/devilinfo/proc/add_soul(datum/mind/soul)
+	var/mob/living/carbon/human/H = owner.current
 	if(soulsOwned.Find(soul))
 		return
 	soulsOwned += soul
+	H.nutrition = NUTRITION_LEVEL_FULL
+	owner.current << "<span class='warning'>You feel satiated as you recieved a new soul."
```
Comment (ChangelingRain) [inline code/game/gamemodes/devil/devilinfo.dm]: Should be receive and not recieved

**Comment by optimumtact** [comment]
Fix that minor spelling issue and I'm happy to merge this

File: code/game/gamemodes/devil/devilinfo.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/gamemodes/devil/devilinfo.dm]: No, received is fine, what is missing however is a ```</span>``` at the end.

Comment (timkoster1) [inline code/game/gamemodes/devil/devilinfo.dm]: Damn I'm stupid, thanks.

**Comment by optimumtact** [comment]
Thank you

**Pull request merged**